src/main/connector/system/Windows/Windows.yaml

```diff
@@ -204,19 +204,49 @@ monitors:
               BytesSentPerSec,
               BytesReceivedPerSec
             FROM Win32_PerfRawData_Tcpip_NetworkInterface
+          computes:
+            - type: duplicateColumn
+              column: 1
+            - type: awk
+              script: 'BEGIN {FS=OFS=";"} {gsub(/[^a-zA-Z0-9]/, " ", $2); print $0}'
+        # Name;IpAddress
+        networkAdapters:
+          type: wmi
+          namespace: root\CIMv2
+          query: >
+            SELECT Description,
+              IPAddress
+            FROM Win32_NetworkAdapterConfiguration

```

Comment:
`Win32_NetworkAdapterConfiguration` lists all network *adapters* (incl. many WAN and virtual network cards), while `Win32_PerfRawData_Tcpip_NetworkInterface` lists only network *interfaces* (Windows makes a slight difference between the 2 concepts).

So: 

1. you either join `Win32_NetworkAdapterConfiguration` with `Win32_PerfRawData_Tcpip_NetworkAdapter`, and provide performance information for all "adapters", including WAN miniports and virtual cards,
2. or join `Win32_PerfRawData_Tcpip_NetworkInterface` (left) with `Win32_NetworkAdapterConfiguration` (right) to make sure only the physical interfaces are listed in the result of the join operation

[Prometheus' `windows_exporter`](https://github.com/prometheus-community/windows_exporter/blob/master/pkg/collector/net/net.go) seems to have chosen the 2nd option, so that's our preferred one to!